beacon-chain/sync/validate_blob.go

```diff
@@ -55,12 +53,15 @@ func (s *Service) validateBlob(ctx context.Context, pid peer.ID, msg *pubsub.Mes
 		return pubsub.ValidationReject, err
 	}
 
-	sBlob, ok := m.(*eth.SignedBlobSidecar)
+	bpb, ok := m.(*eth.BlobSidecar)
 	if !ok {
-		log.WithField("message", m).Error("Message is not of type *eth.SignedBlobSidecar")
+		log.WithField("message", m).Error("Message is not of type *eth.BlobSidecar")
 		return pubsub.ValidationReject, errWrongMessage
 	}
-	blob := sBlob.Message
+	blob, err := blocks.NewROBlob(bpb)
+	if err != nil {
+		return pubsub.ValidationReject, errors.Wrap(err, "roblob conversion failure")
+	}
```

Comment:
I have a PR for these change but we should merge this first and I can deal with conflicts in my PR